src/index.js

```diff
@@ -16,39 +17,81 @@ import './css/base.scss';
 import './images/plane-icon.svg'
 
 const state = {
-  currentUser: new User()
+  currentUser: new User(),
+  trips: null,
+  destinations: null
 }
 
 dom.loadLogin();
+const getUserDetails = (login) => {
+  window.fetch(BASE + TRAVELER_ENDPOINT + login)
+    .then(response => {
+      console.log(response.status)
+      return response.json()
+    })
+    .then(data => data)
```

Comment:
I'm not sure if this is beneficial but you could consider using async function syntax. I was watching some Youtube videos last night about async functions and it did a lot of the same stuff that you do with the fetchs and .thens.  You just specify that the following code is an async function and you use the keyword await which tells the code to wait for the following code to resolve before proceeding.  I liked it because it was much shorter and easier to follow than this format that they taught us in class.

Reply:
Hey Jeremy, thanks for the idea! I've looked into that syntax a little bit and definitely want to experiment with it over interim if not sooner. If I have some extra refactor time for this project, I'll definitely look into it.